packages/terraform-modules-public/gcp/testnet/modules/validator/startup.sh

```diff
@@ -429,6 +427,7 @@ ExecStart=/usr/bin/docker run \\
       --proxy.proxyenodeurlpair=\\"$PROXY_URL\\" \\
   "
 ExecStop=/usr/bin/docker stop -t 60 %N
+ExecStopPost=/usr/bin/docker rm -f %N
```

Comment:
What's this about?

Reply:
It is not related with this PR. It is a bug in the systemd unit that cause you cannot restart the service with systemd tools (systemd stop/restart, or if you restart the instance...) and the container needs to manually removed with dockercli to reconciliate with systemd.
Package `terraform-modules-public` is meant to be a template/skeleton for anyone to run infra connected to celo network.